packages/web/src/content/brand-policy.md

```diff
@@ -12,13 +12,13 @@ The Celo technology and software is open source and free. This means that it is
 
 The protocol package is released under the GNU Lesser General Public License v3, and other packages are released under the Apache License v2.0, unless otherwise specified in the “LICENSE” or similar file at the package's root.
 
-For more details, please read about our source code license and software licensing policy on Github (celo_monorepo - [here](https://github.com/celo-org/celo-monorepo/blob/master/LICENSE); celo_blockchain - [here](https://github.com/celo-org/celo-blockchain/blob/master/COPYING)).
+For more details, please read about our source code license and software licensing policy on Github ([Celo Monorepo](https://github.com/celo-org/celo-monorepo/blob/master/LICENSE) and [Celo Blockchain](https://github.com/celo-org/celo-blockchain/blob/master/COPYING)).
```

Comment:
should these new links open a new tab?

Reply:
ive updated the markdown to component system to make all external links auto open in new tab